spec/features/user_creates_new_session_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+feature "user creates new session" do
+  scenario "successfully" do
+    user = create(:user)
+    visit login_path
+
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+    click_on I18n.t("sessions.new.submit")
+
+    expect(page).to have_text(I18n.t("users.show.welcome", name: user.name))
+    expect(page).
+      to have_text(I18n.
+        t("flash.actions.create.notice", resource_name: "Session"))
+  end
+
+  scenario "logs out" do
+    user = create(:user)
+    visit login_path
+
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+    click_on I18n.t("sessions.new.submit")
+
```

Comment:
Same comments

Reply:
Also, you might have a helper method to log in the user.

If not, and yo need to manually log the user every time, a suggestion to add a helper method in `spec/support/features/login_helper_methods.rb`

Reply:
The login helper method is awesome! Thanks Elle.
Done in sha: e47bdda